wordle-web/src/scripts/__tests__/wordsService.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, it, expect } from 'vitest'
+import { WordsService } from '../wordsService'
+import { Word } from '../word'
+
+describe('wordsService', () => {
+  it('will test a value that should return multiple available words', () => {
+    const word = new Word('ball')
+    const expected = ['balls', 'bally']
+    const actual = WordsService.validWords(word.text)
+    expect(expected).toStrictEqual(actual)
+  })
+
+  it('will test a value that should return one available word', () => {
+    const word = new Word('pizz')
+    const expected = ['pizza']
+    const actual = WordsService.validWords(word.text)
+    expect(expected).toStrictEqual(actual)
+  })
+
+  it('will test a value that should return multiple available words', () => {
```

Comment:
I don't think this line describes what this test actually is doing. Looks like you copy/pasted the format of the first test and possibly forgot to edit this line. 
```suggestion
  it('will test a value that should return no available words', () => {
```